package.json

```diff
@@ -1,43 +1,43 @@
 {
-	"name": "cooper-csesg",
-	"version": "0.0.2",
-	"description": "WIP boilerplate for Cooper Union Computer Science and Entrepreneurship for Social Good course",
-	"private": true,
-	"scripts": {
-				"build-css": "node-sass-chokidar src/ -o src/",
-				"watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
-				"start-js": "react-scripts start",
-				"build-js": "react-scripts build",
-				"start": "npm-run-all -p watch-css start-js",
-				"build": "npm-run-all build-css build-js",
-				"test": "react-scripts test --env=jsdom",
-				"eject": "react-scripts eject"
-	},
-	"dependencies": {
-				"@material-ui/core": "^1.4.0",
-				"@material-ui/icons": "^1.1.0",
-				"firebase": "^5.2.0",
-				"lodash": "^4.17.10",
-				"material-ui": "^0.20.1",
-				"node-sass-chokidar": "^1.3.3",
-				"normalize.css": "^8.0.0",
-				"npm-run-all": "^4.1.3",
-				"react": "^16.4.1",
-				"react-dnd": "^6.0.0",
-				"react-dnd-html5-backend": "^6.0.0",
-				"react-dom": "^16.4.1",
-				"react-redux": "^5.0.7",
-				"react-redux-firebase": "^2.1.6",
-				"react-router": "^4.3.1",
-				"react-router-dom": "^4.3.1",
-				"react-scripts": "1.1.4",
-				"recompose": "^0.27.1",
-				"redux": "^4.0.0",
-				"redux-auth-wrapper": "^2.0.3",
-				"redux-form": "^7.4.2",
-				"redux-form-material-ui": "^5.0.0-beta.3",
-				"redux-thunk": "^2.3.0",
-				"whatwg-fetch": "^2.0.4"
-	},
-	"devDependencies": {}
+			"name": "cooper-csesg",
+			"version": "0.0.2",
+			"description": "WIP boilerplate for Cooper Union Computer Science and Entrepreneurship for Social Good course",
+			"private": true,
+			"scripts": {
+						"build-css": "node-sass-chokidar src/ -o src/",
+						"watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
+						"start-js": "react-scripts start",
+						"build-js": "react-scripts build",
+						"start": "npm-run-all -p watch-css start-js",
+						"build": "npm-run-all build-css build-js",
+						"test": "react-scripts test --env=jsdom",
+						"eject": "react-scripts eject"
+			},
+			"dependencies": {
+						"@material-ui/core": "^1.4.0",
+						"@material-ui/icons": "^1.1.0",
+						"firebase": "^5.2.0",
+						"lodash": "^4.17.10",
+						"material-ui": "^0.20.1",
+						"node-sass-chokidar": "^1.3.3",
+						"normalize.css": "^8.0.0",
+						"npm-run-all": "^4.1.3",
+						"react": "^16.4.1",
+						"react-dnd": "^6.0.0",
+						"react-dnd-html5-backend": "^6.0.0",
+						"react-dom": "^16.4.1",
+						"react-redux": "^5.0.7",
+						"react-redux-firebase": "^2.1.6",
+						"react-router": "^4.3.1",
+						"react-router-dom": "^4.3.1",
+						"react-scripts": "1.1.4",
+						"recompose": "^0.27.1",
+						"redux": "^4.0.0",
+						"redux-auth-wrapper": "^2.0.3",
+						"redux-form": "^7.4.2",
+						"redux-form-material-ui": "^5.0.0-beta.3",
+						"redux-thunk": "^2.3.0",
+						"whatwg-fetch": "^2.0.4"
+			},
+			"devDependencies": {}
 }
```

Comment:
This file seems like only whitespace has been added. If you do a `git diff <file_name>` locally, and find such files, do not `git add` them to your commit.